execution/evm/vm.go

```diff
@@ -955,11 +999,13 @@ func (vm *VM) call(caller acm.Account, callee acm.MutableAccount, code, input []
 						return nil, firstErr(callErr, ErrUnknownAddress)
 					}
 					ret, callErr = vm.Call(callee, callee, acc.Code(), args, value, &gasLimit)
+					vm.setReturnDataBuffer(ret)
```

Comment:
remove these in favour of setting once after this block. It's simpler and clearer and currently we are not capturing return data from native contracts, which we want to do.